config/locales/en/whats_new.yml

```diff
@@ -0,0 +1,47 @@
+en:
```

Comment:
I've just noticed we seem to be creating new conventions with this locale file - it seems different to all the other ones that exist. Have the been any discussions about the organisation?

To follow the [Rails convention for key structure](https://guides.rubyonrails.org/i18n.html#lazy-lookup) the translations in this file should be under:

```
en:
  admin:
    whats_new:
      index:
         # your data
```

Which would mean this file would exist at /config/locales/en/admin/whats_new/index.yml

Reply:
done